doc/content/applications/se-vanilla/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Sensor Edition: vanilla application"
+description: ""
+weight: -2
+distributions: null
+---
+
+{{% gnse %}} comes pre-programmed with a vanilla application that showcases the device functionality and capabilities.
+
+The application leverages a combination of the on-board components such as LEDs, button, buzzer, accelerometer, secure element, temperature and humidity sensor.
+
+<!--more-->
+
+## Claiming your device
+
+The vanilla application uses the on-board secure element to join the LoRaWAN network using over the air activation (OTAA) method.
+
+You can claim your device on the The Things Stack cloud by following this [guide](https://www.thethingsindustries.com/docs/devices/device-claiming/claim-devices/).
+
+## Powering up your device
+
+After claiming, you can power up the device using two standard AA batteries.
+
+If the device powers up correctly, you will see the red, green and blue LEDs blinking thus indicating successful boot-up.
+
+## Joining the network
+
+The device will attempt to join LoRaWAN network using OTAA indicated by the blinking red led.
+
+When the device joins successfully, the buzzer will ring briefly the red LED will stop blinking.
+
+## Default uplinks and events
+
+After a successful join, the device will periodically transmit two types of messages.
+
+1. A heartbeat message on FPort 1 every 25 seconds
+2. A Temperature and Humidity reading on FPort 2 every 1 minute
+
+You can also press the button to transmit an uplink with the heartbeat message.
+
+The device will blink the green LED with each successful uplink.
+
+## Heartbeat
+
+The heartbeat uplink message sent by the device on FPort 1 contains the device battery voltage and the firmware version.
+
+## Temperature and Humidity
+
+The temperature and humidity uplink message sent on FPort 2 contains the temperature in Celsius and the humidity in percentage.
+
+## Accelerometer
+
+The accelerometer is disabled by default and can be enabled via a downlink on FPort 3 to detect shake and free-fall events.
+When an accelerometer event occurs (shake or free-fall), the device will transmit an uplink with the temperature and humidity readings.
+
+{{< note "The accelerometer should be disabled when it's not used to conserve energy" />}}
+
+## Button
+
+The button is enabled by default and can be disabled or enabled via a downlink on FPort 4.
+When the button is enabled, pressing the button will send a heartbeat uplink message.
+
+## Buzzer
+
+The buzzer is disabled by default can be enabled via a downlink on FPort 5.
+The buzzer can ring briefly in two tones, warning and danger.
```

Comment:
```suggestion
The buzzer is disabled by default and it can be enabled via a downlink on FPort 5.

The buzzer can briefly ring in two tones, indicating warning or danger.
```